bodyfetcher.py

```diff
@@ -208,10 +241,37 @@ def make_api_call_for_site(self, site):
                 owner_rep = 0
             q_id = str(post["question_id"])
 
-            is_spam, reason, why = check_if_spam(title, body, owner_name, owner_link, site, q_id, False, False, owner_rep, post_score)
+            is_spam, reason, why = check_if_spam(
```

Comment:
Making each argument get its own line is... really ugly. I believe the Google Style Guide agrees with me here: https://google.github.io/styleguide/pyguide.html#Line_length